tests/helpers/jasmine-hooks.conf.js

```diff
@@ -0,0 +1,7 @@
+import { config as baseConfig } from './config.js'
+
+export const config = Object.assign({}, baseConfig, {
+    beforeTest() {
+        pending()
```

Comment:
Can we add a comment that this is a global function defined by Jasmine?

Reply:
```suggestion
        // this is a global function defined by Jasmine
        pending()
```

Reply:
done